brd-android/fabriik-common/src/main/java/com/fabriik/common/ui/dialog/FabriikGenericDialog.kt

```diff
@@ -63,11 +63,21 @@ class FabriikGenericDialog : DialogFragment() {
     }
 
     private fun setupDescription(view: TextView) {
-        val description = if (args.descriptionRes != null) getString(args.descriptionRes!!) else args.description
+        val description = if (args.descriptionRes != null) getStringRes() else args.description
         view.text = description
         view.isVisible = description != null
     }
 
+    private fun getStringRes(): String {
```

Comment:
This is a bad solution, we need to allow undefined number of arguments. You can do something similar:

```suggestion
    private fun getStringRes(): String {
        val messageArgs = args.messageArgs?.toTypedArray() ?: emptyArray()
        return getString(args.descriptionRes!!, *messageArgs)
    }
```